data-raw/manifest.R

```diff
@@ -1,11 +1,40 @@
 library(data.table)
 
-# -------------------------------- EPIC chip manifest
+# -------------------------------- EPIC V2 chip manifest
 
-### CSV contains both 'normal' and control probes. Create two separate tables for them (split at line 865927)
+### NEW EPIC array added by Costanza L. Vallerga
+### CSV contains both 'normal' and control probes. Create two separate tables for them (split at line 937055)
+
+manifest_epic_v2 = fread("EPIC-8v2-0_A1.csv",skip="IlmnID",header=TRUE,nrows=937055,integer64="character",sep=",",sep2=";")
+
+manifest_epic_v2 = manifest_epic_v2[,list(
+     probe_id=IlmnID
+    ,addressU=as.integer(AddressA_ID)
+    ,addressM=as.integer(AddressB_ID)
+    ,channel=Color_Channel
+    ,next_base=Next_Base
```

Comment:
```suggestion
    ,next_base=Next_Base
    ,probe_type=Probe_Type
```

Column exist in manifest already